packages/scheduler-prisma/src/helpers.ts

```diff
@@ -17,39 +17,88 @@ const SIX_MONTHS_IN_MS = 15778476000;
 export const isHealthy = (heartbeat: number, timeout: number) =>
   new Date().getTime() - timeout < heartbeat;
 
+const getValidRule = (recurrence: string, timezone?: string) => {
+  const isICalRule = recurrence.includes('DTSTART');
+  if (isICalRule) return recurrence;
+
+  let derivedTimezone = timezone ?? 'Australia/Sydney';
+  const rule = recurrence
+    .split(';')
+    .filter(b => {
+      const [key, value] = b.split('=');
+      if (key === 'TZID') {
+        derivedTimezone = value;
+      }
+      return key !== 'TZID';
+    })
+    .join(';');
+
+  const timeISO8601 = moment().tz(derivedTimezone).format('YYYYMMDDTHHmmss');
+  return `DTSTART;TZID=${derivedTimezone}:${timeISO8601}\nRRULE:${rule}`;
+};
+
 // interval should be iCal String.
-export const computeNextRunAt = (
+export const computeNextRun = (
   interval: string,
-  timezone = 'UTC'
+  {
```

Comment:
I assume we already did a sanity check on this usage cause I'm a bit concern with this function signature change but I believe we only use the sequelize version for job migrations in OM so I think this is fine 👍